projects/distributed-tracing/src/shared/dashboard/widgets/trace-detail/data/trace-detail-data-source.model.ts

```diff
@@ -21,21 +21,29 @@ export class TraceDetailDataSourceModel extends GraphQlDataSourceModel<TraceDeta
   })
   public trace!: Trace;
 
+  @ModelProperty({
+    key: 'start-time',
+    required: false,
+    type: UNKNOWN_PROPERTY.type
```

Comment:
why unknown?

Reply:
Because i think we can't put union types in model properties. This field can be string, number or undefined

Reply:
But why not just say number (or string)? Given that we can't be as broad as we want (the code would work with string | number), it seems better to be narrower and force the value provider to match (vs having an overly broad type where some values would result in an error)

Reply:
Surely we could do that. The unknown pattern was there (may be i used it initially for a valid reason). With current design, i agree we could just force it to be of a particular type.